ports/stm32/usbd_cdc_interface.c

```diff
@@ -236,6 +236,25 @@ void HAL_PCD_SOFCallback(PCD_HandleTypeDef *hpcd) {
     }
 }
 
+bool usbd_cdc_rx_buffer_any(usbd_cdc_itf_t *cdc) {
```

Comment:
This isn't a good name for C function. There's no need to bring Python's high-level API convention down to low-level, much-detail level of C.

Reply:
Well, natural way is to name like you did for a variable in this patch - using "full" and ! condition. If it's even more complex than that (given `remaining > (CDC_DATA_MAX_PACKET_SIZE*2)`), then just using "enough_free", albeit long, would be at least truthful.